src/implementations/vanilla/components/message/_message.scss

```diff
@@ -31,26 +48,31 @@ $_close-color: map.get(theme.$color, 'black-100');
 
 .ecl-message__title {
   color: $_title-color;
-  font: map.get(theme.$font-prolonged, 'm');
+  font: $_title-font;
   font-weight: bold;
-  margin-bottom: map.get(theme.$spacing, 'xs');
+  margin-bottom: $_title-spacing;
   max-width: 80ch;
 }
 
 .ecl-message__description {
   color: $_description-color;
-  font: map.get(theme.$font, 'xs');
-  margin: 0;
+  font: map.get(theme.$font-prolonged, 'm');
+  margin: 0 0 map.get(theme.$spacing, 'xs');
   max-width: 80ch;
   padding-bottom: map.get(theme.$spacing, 'xl'); // Extra spacing for IE
 }
 
 .ecl-message__close {
   bottom: map.get(theme.$spacing, 'm') - map.get(theme.$spacing, 's');
   color: $_close-color;
-  font: map.get(theme.$font, 's');
+  font: map.get(theme.$font, 'm');
+  font-weight: $_close-font-weight;
   position: absolute;
   right: 0;
+  // stylelint-disable-next-line plugin/selector-bem-pattern
+  .ecl-icon {
+    fill: map.get(theme.$color, 'blue-100');
```

Comment:
ecl-icon is using `fill: currentColor;`, which is also blue-100, so I don't think that we need to override it here